forge-gui/res/cardsfolder/upcoming/the_tenth_doctor.txt

```diff
@@ -0,0 +1,12 @@
+Name:The Tenth Doctor
+ManaCost:3 U R
+Types:Legendary Creature Time Lord Doctor
+PT:3/5
+T:Mode$ AttackersDeclared | AttackingPlayer$ You | Execute$ TrigExile | TriggerZones$ Battlefield | TriggerDescription$ Allons-y! — Whenever you attack, exile cards from the top of your library until you exile a nonland card. Put three time counters on it. If it doesn't have suspend, it gains suspend.
+SVar:TrigExile:DB$ DigUntil | Valid$ Card.nonLand | ValidDescription$ nonland | FoundDestination$ Exile | RevealedDestination$ Exile | RememberFound$ True | SubAbility$ DBPutCounter
+SVar:DBPutCounter:DB$ PutCounter | Defined$ Remembered | CounterType$ TIME | CounterNum$ 3 | SubAbility$ GiveSuspend
+SVar:GiveSuspend:DB$ Pump | Defined$ Remembered | KW$ Suspend | PumpZone$ Exile | Duration$ Permanent | SubAbility$ DBCleanup
```

Comment:
same problem

Reply:
you could have just written `Defined$ Remembered.withoutSuspend`
(one of the smart ways to combine Defined + Valid syntax)

Reply:
you're still using the wrong effect/param so you can't use the syntax

Reply:
Ah yeah defined with pumpall is for players